students/KyleBarry/session04/trigrammify.py

```diff
@@ -0,0 +1,60 @@
+import random
+import requests
+from bs4 import BeautifulSoup
+
+
+def get_text(url): 
+    """ 
+    Get text data from url of gutenberg project so as to not have it within code
+    """
+    page = requests.get(url)
+    soup = BeautifulSoup(page.content, "lxml")
+    text = soup.find_all("p")
+    texter = [i.text for i in text]
+    words = "".join(texter).split()
+    # Pass parsed words of book in list form to trigram dictionary maker
+    make_trigrams(words)
```

Comment:
better to return the words here, and pass that on outside this function.

Also -- maybe have this function cache the book, or the words, on disk, so you don't have to re-download every time!